sdk/generic-sdk-core/generic-core-http-okhttp/src/main/java/com/generic/core/http/okhttp/implementation/OkHttpToGenericCoreHttpHeadersWrapper.java

```diff
@@ -18,13 +18,13 @@
  * (as these headers are the result of an OkHttp response), we do so in any case, given the additional implementation
  * cost is minimal.
  */
-public final class OkHttpToAzureCoreHttpHeadersWrapper extends com.generic.core.models.Headers {
+public final class OkHttpToGenericCoreHttpHeadersWrapper extends com.generic.core.models.Headers {
```

Comment:
```suggestion
public final class OkHttpToCoreHttpHeadersWrapper extends com.generic.core.models.Headers {
```

Reply:
I'll change this ina  separate PR, for now I've reverted unrelated changes.